src/components/app/App.tsx

```diff
@@ -49,7 +49,7 @@ export class Evergreen extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props);
     this.state = {
-      APIClient: rest.EvergreenClient("", "", "", "", true)
+      APIClient: rest.EvergreenClient("https://evergreen.mongodb.com/api", "https://evergreen.mongodb.com", "", "")
```

Comment:
This shouldn't be populated in the constructor (and if it did it should come from the config)

Reply:
You're right- I just had this in for convenience's sake while I was testing. I'll go ahead and remove it.